functions/src/utils/parsers.ts

```diff
@@ -0,0 +1,46 @@
+import { format } from "date-fns";
+
+export const EMPTY_VALUE = "-";
+
+interface IAudioDescription {
+  beginning: string; // h:mm:ss
+  ending: string; // h:mm:ss
+  type: string;
+  description: string;
+}
+
+/**
+ * Used for Unwanted Parts and Sound Issues to create multi-line comments
+ *
+ */
+export function formatMultilineComment(audioDescriptionList: IAudioDescription[]) {
+  if (!audioDescriptionList || !audioDescriptionList.length) {
+    return "-";
+  }
+  let multiline = "";
+  audioDescriptionList.forEach((elem: IAudioDescription, index: number) => {
+    multiline = multiline
+    + `${elem.beginning}-${elem.ending}:${elem.type} -- ${elem.description}`
+    + ((audioDescriptionList.length === (index + 1)) ? "" : "\n");
+  });
+  return multiline;
+}
+
+export function createUpdateLink(token: string): string {
+  return `http://purebhakti.info/audioseva/form/sound-quality-report?token=${token}`;
```

Comment:
This particular function is needed only in SQR module, why not to move it there?

Reply:
Remove this from parsers! It's SQR-specific.